utils/pod/pod.go

```diff
@@ -104,16 +104,20 @@ func checkContainerExitCode(containerId string) (int, error) {
 }
 
 //get docker health check logs
-func PrintHealthCheckLogs(containerId string) error {
-	out, err := waitUtil.RetryCmd(config.GetMaxRetry(), exec.Command("docker", "inspect",
-		"--format='{{json .State.Health}}'",
-		containerId))
+func PrintInspectDetail(containerId string) error {
+	dceLog := config.CreateFileAppendMode(types.DCE_OUT)
+	dceErr := config.CreateFileAppendMode(types.DCE_ERR)
+	cmd := exec.Command("docker", "inspect",
+		containerId)
+	cmd.Stdout = dceLog
+	cmd.Stderr = dceErr
+	out, err := waitUtil.RetryCmd(config.GetMaxRetry(), cmd)
 
 	if err != nil {
 		log.Println("Error inspecting container for health check details :", err)
 		return err
 	}
-	log.Println("Health Check inspect Logs: ", string(out))
+	fmt.Println("Inspect Logs: ", string(out))
```

Comment:
use log.Println

Reply:
it needs to be fmt which easily help to convert character '\n' to a new line. So that we can get pretty print json. Otherwise, the output will be just showing character '\n' like following
```
{a\n, b\n, c\n}
```

Reply:
fmt goes to stdout, logrus default goes to stderr, escaping `\n` is the desired behavior for machine to parse json format log. Though if your use case is just manually inspect log this works fine

- https://golang.org/src/fmt/print.go?s=7953:8002#L264
- https://github.com/sirupsen/logrus/blob/c078b1e43f58d563c74cebe63c85789e76ddb627/logger.go#L68:6